src/main/java/com/github/calebwhiting/runelite/plugins/actionprogress/detect/WintertodtDetector.java

```diff
@@ -36,16 +35,12 @@ public class WintertodtDetector extends ActionDetector
 
 	private static final String[] INTERRUPT_MESSAGES = {
 			"The cold of the Wintertodt seeps into your bones.", "The freezing cold attack",
-			"The brazier is broken and shrapnel damages you.", "The brazier has gone out."
+			"The brazier is broken and shrapnel damages you.", "The brazier has gone out.",
```

Comment:
Would the interrupt trigger if the minigame fail?

Reply:
As we discussed in discord, no failure in wintertodt :)